ytypes/util_test.go

```diff
@@ -304,7 +305,7 @@ func TestStringToKeyType(t *testing.T) {
 	listSchema := &yang.Entry{
 		Name:     "struct-key-list",
 		Kind:     yang.DirectoryEntry,
-		ListAttr: &yang.ListAttr{MinElements: &yang.Value{Name: "0"}},
+		ListAttr: util.NewListAttr(),
```

Comment:
I'd suggest putting this in `testutil` rather than `util`, since it's not quite general-purpose enough for a utility is it (we might expect that to have a way to specify min/max values). If we decide not to put it in `testutil` then at least it should be named something like `NewDefaultListAttr`.

Reply:
I'm waiting for https://github.com/openconfig/goyang/pull/147 to be merged, and to use the function from goyang instead.